certified-connectors/Azure Communication Services Identity/apiProperties.json

```diff
@@ -0,0 +1,29 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "api_key": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "Connection String",
+          "description": "Azure Communication Services Connection String.",
+          "tooltip": "Find your connection string in your Azure Communication Services resource in Azure Portal in \"Keys\".",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#007ee5",
+    "scriptOperations": [
```

Comment:
@lei-msft can you please provide a `script.csx` for this section if possible? One of the checks fails because of that.